lms/djangoapps/instructor/services.py

```diff
@@ -0,0 +1,45 @@
+"""
+Implementation of "courseware" service
+"""
+
+import logging
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from courseware.models import StudentModule
+from instructor.views.tools import get_student_from_identifier
+import instructor.enrollment as enrollment
+
+
+log = logging.getLogger(__name__)
+
+
+class CoursewareService(object):
+    """
+    Courseware service for deleting student attempt
+    """
```

Comment:
Let's extend this comment to mention that edx_proctoring is the only consumer of this service for now. Also explain why we need to do this (i.e. Dependency Injection)